### ISSUE TITLE
Gangs

### ISSUE THREAD

**Issue by Dru184** [AaronWeber53/SRC-RP-Discussions#161]
So when it comes to gangs, and later on having them WhiteListed, I think we need to keep in mind the way they treat PD as well. How much is RP and how much is just shooting at PD. I personally wouldn't like a gang that just shoots at PD to get whitelisted. This will "force" them to have good scenes and RP with cops, as criminals need to be stealthy about their affairs and good relations with PD can take them a long way.

Was thinking of having a discord or at least a few channels where high command PD and high gang members and their leaders can send us footage or any concerns they might have so we can then discuss it in city. A bridge between Crims and PD, give them a voice and let them know that we are willing to listen and work things out so the city isn't a battlefield. This will provide more RP and less tickets.

Gang strikes.

If a gang gets disbanded, they are not allowed to make another gang under a different name, but if you look at it, it's the same exact people, just a different leader. I've seen it happen a few times when disbanded gangs make a new one and they are let to do so because they have a new leader so things will change. They don't. They are just cheating the system. If 60% of the people in this new gang are new people, then I think that should be fine, but not have 90% same people as the ones from the gang that got disbanded. Things won't change.

**Comment by AaronWeber53** [comment]
Yeah I do agree that we probably will have rules for gangs and if they ever break the rules under the discretion of the Owners we will remove the gang which means they lose all of the benefits (gang dealer reputation and any future features) and money/items they stored under the gang name.
